frontend/src/bbkit/BBModal.vue

```diff
@@ -11,8 +11,28 @@
     >
       <div class="relative -mt-4 -ml-4 flex items-center justify-between">
         <div class="ml-4 text-xl text-main">
-          {{ title }}
-          <div v-if="subtitle" class="text-sm text-control whitespace-nowrap">
+          <component
+            :is="overrides.title"
+            v-if="typeof overrides.title === 'function'"
```

Comment:
Maybe we could use `computed` to check types in `setup`, so the rendering will looks normally in `template`. wdyt?